development/compiling/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _compiling:
+
```

Comment:
Index files are already given a name below. This one is "toc-devel-compiling". You should use that to reference this document rather than adding a second name.